spk/minio/src/service-setup.sh

```diff
@@ -0,0 +1,39 @@
+# Setup environment
+PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
+
```

Comment:
Add

```
GROUP="sc-minio"
```

for `spk` to create the correct group.

Reply:
@fgma That should do the trick, no?

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/3689/commits/0dfdb3902a373eba35904028c813ea2099625b89

Reply:
Just tested it, didn't change anything. Maybe @hgy59 or @ymartin59 can shed some light here on why the group `sc-minio` is not created.

Reply:
If it did ever work for you the only change I can imagine might be https://github.com/SynoCommunity/spksrc/pull/3689/commits/6528efca4ad996e3c3d613188c4d4655630d0ab4.

Can you check if reverting back "SERVICE_WIZARD_GROUP = wizard_group" fixes the problem?

Reply:
@hgy59 @ymartin59 Can anyone give me a hint on solving this? What is the proper way to ensure the group is created without using SERVICE_WIZARD_GROUP?

Reply:
GROUP="sc-minio" here in "service-setup.sh" should work as it replaces generated variable from https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.mk#L87

Reply:
@fgma I just verified with Syncthing and it works perfectly there. Somehow your commit did not make it into your branch? Because when I pull your repo and checkout branch minio the change is not there... Didn't verify the source before. So what I noted earlier works as confirmed by @ymartin59.

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/3689/commits/175beaee8b5b65cd0246c7b0ec6699b40afee919. @alexanderharm  Please test it.

Reply:
All fine now! Thanks a lot, I already run this in production.

Reply:
Thanks!